contributor-guide/modules/ROOT/pages/contributors-faq.adoc

```diff
@@ -0,0 +1,129 @@
+////
+Copyright (c) 2024 The C++ Alliance, Inc. (https://cppalliance.org)
+
+Distributed under the Boost Software License, Version 1.0. (See accompanying
+file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+Official repository: https://github.com/boostorg/website-v2-docs
+////
+= Contributors Frequently Asked Questions
+:navtitle: Contributors FAQ
+
+This section contains answers to the common questions that new contributors to Boost often have.
+
+== Artificial Intelligence
```

Comment:
Anything about artificial intelligence is not a "common question that new contributors to Boost often have".

That also goes way beyond questions on Modular Boost that came from email exchanges, and questions on creating new libraries. (The PR description)

Reply:
OK, I'll remove this section for now and find a better location for it - if there is one.